pkgs/development/compilers/factor-lang/unwrapped.nix

```
@@ -0,0 +1,107 @@
+{ lib
+, stdenv
+, fetchurl
+, makeWrapper
+, curl
+, git
+, ncurses
+, tzdata
+, unzip
+}:
+let
+  rev = "e10b64dbc53a8583098e73580a1eb9ff4ce0c709";
+
+in
+stdenv.mkDerivation (finalArgs: {
+  pname = "factor-lang";
+  version = "0.99";
+
+  src = fetchurl {
+    url =
+      "https://downloads.factorcode.org/releases/${finalArgs.version}/factor-src-${finalArgs.version}.zip";
+    sha256 = "f5626bb3119bd77de9ac3392fdbe188bffc26557fab3ea34f7ca21e372a8443e";
+  };
+
+  patches = [
+    # Use full path to image while bootstrapping
+    ./staging-command-line-0.99-pre.patch
+    # Point work vocabulary root to a writable location
+    ./workdir-0.99-pre.patch
+    # Patch hard-coded FHS paths
+    ./adjust-paths-in-unit-tests.patch
+    # Avoid using /sbin/ldconfig
+    ./ld.so.cache-from-env.patch
+  ];
+
+  nativeBuildInputs = [ git makeWrapper curl unzip ];
+
+  postPatch = ''
+    sed -ie '4i GIT_LABEL = heads/master-${finalArgs.passthru.rev}' GNUmakefile
```

Having rev here breaks the auto updater. We can just mock it and make updates easy.

---

we can just replace it with 0s
```suggestion
    sed -ie '4i GIT_LABEL = heads/master-${lib.fakeSha256}' GNUmakefile
```